src/amo/css/OverallRating.scss

```diff
@@ -1,29 +1,33 @@
+
 .OverallRating-choices {
   display: flex;
   justify-content: center;
 }
 
+$star-size: 46px;
+
 .OverallRating-choice {
-  font-size: 0.8em;
   font-weight: bold;
   cursor: pointer;
-  text-align: center;
-  padding-top: 90px;
-  width: 100px;
-  height: 100px;
-  background-size: 85px;
+  padding-top: 80px;
+  width: $star-size + 10px;
+  height: $star-size + 10px;
+  background-size: $star-size;
   background-position: center;
   background-repeat: no-repeat;
+  background-image: url('../img/ratings/big-star.svg');
```

Comment:
In general I think we've been using `background: ` shorthand, but since it all gets minified anyway I don't care much.